src/Controller/ProblemController.php

```diff
@@ -209,9 +209,9 @@ public function problemComment($participantId, $problemId, Request $request, Log
         }
         if ($problemId) {
```

Comment:
Could we instead remove this check? The problemId has to be set, since it's part of the route, and doesn't have a default value. If you remove that if, then $problem will always be set to the result of the `findOneBy` and we can continue using the same check as before.